cve-suppressions.xml

```diff
@@ -1,67 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-    <suppress>
```

Comment:
Thanks for fixing this list! Was in my plans to do it someday!

Reply:
My pleasure! I really like that with this upgrade, it is now very minimalistic :)